package.json

```diff
@@ -1,43 +1,47 @@
 {
 	"name": "jstreegrid",
 	"description": "grid plugin for jstree",
-	"version": "3.8.2",
+	"version": "3.9.1",
```

Comment:
Why 3.9.1? Why not 3.8.3? According to semver, this has no API change so should remain same minor version.